### PR TITLE
Sscs 9746 confirm panel composition event

### PR DESCRIPTION
**Pull request by gmikerlemos**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-9746

### Change description ###

Child Support: Confirm Panel Composition Event

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json [hunk 1]
```diff
@@ -22,5 +22,7 @@
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequest", "UserRole": "caseworker-sscs", "CRUD": "R"},
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs-superuser", "CRUD": "R"},
-  {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs", "CRUD": "R"}
+  {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs", "CRUD": "R"},
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseEventID": "confirmPanelComposition", "UserRole": "caseworker-sscs-judge", "CRUD": "CRUD"},
```
Comment (ryandac) [inline benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:26]: think there should be a R option for user role caseworker-sscs

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
   {"LiveFrom": "16/09/2019", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "pcqId", "TabFieldDisplayOrder": 31},
-  {"LiveFrom": "01/10/2021", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "summaryConfidentialityRequired", "FieldShowCondition": "appeal.appellant.confidentialityRequired!=\"\"", "TabFieldDisplayOrder": 32}
+  {"LiveFrom": "01/10/2021", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "summaryConfidentialityRequired", "FieldShowCondition": "appeal.appellant.confidentialityRequired!=\"\"", "TabFieldDisplayOrder": 32},
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "isFqpmRequired", "TabFieldDisplayOrder": 38, "FieldShowCondition": "isFqpmRequired!=\"\""},
```
Comment (ryandac) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json:6]: This line doesn't belong here please move it down one line to make the job the person releasing it a bit easier.

**Review by ryandac — CHANGES_REQUESTED** [review]

File: benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json (hunk 1 above)
Comment (gmikerlemos) [reply benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:26]: as per AC only superuser and judge can access the event

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json (hunk 2 above)
Comment (gmikerlemos) [reply benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json:6]: all lines at the bottom are grouped per feature, can't see why you think it doesn't belong there.

File: benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json (hunk 1 above)
Comment (jackmaloney) [reply benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:26]: But if you don't add Read permission, it won't show in history of the case which creates confusion as they can't see everything that has happened on the case. We normally add Read permission as a minimum for all users for all events (e.g.  caseworker-sscs) unless there is a good reason not to

Comment (gmikerlemos) [reply benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:27]: Ok, added the R permission

**Review by jackmaloney — APPROVED** [review]

**Review by ryandac — APPROVED** [review]

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json (hunk 2 above)
Comment (ryandac) [reply benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json]: It was not grouped per tab id. It's not very interesting to explain. Before this project we had to update a spreadsheet that regularly got into a mess.

**Pull request merged**